### ISSUE TITLE
explicitly say the gem doesn't support Rails 2.3

### ISSUE THREAD

**Issue by Nowaker** [tkadauke/i18n_tools#3]
- add it to readme
- `spec.add_runtime_denendency 'rails', '>= 3.0'`

**Comment by Nowaker** [comment]
Or bring back Rails 2.3 support:
https://github.com/tkadauke/i18n_tools/commit/0e00263e76bf8ace9564407a303c42aa9430506e#commitcomment-5397920

**Comment by tkadauke** [comment]
I use the gem with both Rails 2.3 and Ruby 1.8.7 ... what seems to be the problem?

**Comment by Nowaker** [comment]
@tkadauke This is the problem: https://github.com/tkadauke/i18n_tools/commit/0e00263e76bf8ace9564407a303c42aa9430506e#commitcomment-5397920

**Comment by Nowaker** [comment]
> I use the gem with both Rails 2.3

Not quite. ;) https://github.com/tkadauke/i18n_tools/commit/0e00263e76bf8ace9564407a303c42aa9430506e#commitcomment-5960228

```
>> Rails.version
=> "2.3.18"
>> Rails::Railtie
NameError: uninitialized constant Rails::Railtie
        from /home/nowaker/.rvm/gems/ruby-1.9.3-p545/bundler/gems/rails-3a3c18317943/activesupport/lib/active_support/dependencies.rb:466:in `rescue in load_missing_constant'
        from /home/nowaker/.rvm/gems/ruby-1.9.3-p545/bundler/gems/rails-3a3c18317943/activesupport/lib/active_support/dependencies.rb:462:in `load_missing_constant'
        from /home/nowaker/.rvm/gems/ruby-1.9.3-p545/bundler/gems/rails-3a3c18317943/activesupport/lib/active_support/dependencies.rb:106:in `const_missing_with_dependencies'
        from (irb):4
        from /home/nowaker/.rvm/rubies/ruby-1.9.3-p545/bin/irb:12:in `<main>'
```

**Comment by tkadauke** [comment]
Ok, just released a new version (0.1.6) that works with Rails 2.3.18.

Thanks for the bug report! :-)

**Comment by Nowaker** [comment]
Thanks.
